src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalCompletionService.ts

```diff
@@ -204,12 +205,7 @@ export class TerminalCompletionService extends Disposable implements ITerminalCo
 
 		const resourceCompletions: ITerminalCompletion[] = [];
 		const endsWithSpace = promptValue.substring(0, cursorPosition).endsWith(' ');
```

Comment:
`promptValue.substring(0, cursorPosition)` could be pulled into a `cursorPrefix` var?